### DISCUSSION TITLE
Longhorn as a standalone storage cluster solution

### DISCUSSION THREAD

**Discussion by inandakumar** [longhorn/longhorn#3722] [Q&A]
Hi All, I have a bunch of servers unused in our data center. We are currently in need of remote storage for few of our applications.
I am wondering if we could spin up Longhorn on these set of boxes and form a storage cluster which serves remote storage to applications running outside the cluster. Is this possible and/or supported by Longhorn?

**Comment by jenting** [comment]
In general no, Longhorn is dedicated for Kubernetes in cluster storage provider.
Although Longhorn could expose iscsi target but we didn't provide the best practices for this scenario.
